forc-plugins/forc-client/src/util/tx.rs

```diff
@@ -156,7 +160,9 @@ impl<Tx: Buildable + SerializableVec + field::Witnesses + Send> TransactionBuild
                         let accepted = ask_user_yes_no_question(&question)?;
                         if accepted {
                             new_wallet_cli(&wallet_path)?;
-                            println!("Wallet created successfully.")
+                            println!("Wallet created successfully. Deriving first account");
+                            // Derive first account for the fresh wallet we created.
+                            new_at_index_cli(&wallet_path, 0)?;
```

Comment:
Should we have a success message after the account is derived?

Reply:
Currently it looks like the following:
```
Wallet created successfully. Deriving first account
Please enter your wallet password to derive account 0: 
Wallet address: fuel1dy60cz4kzxxzexdaqkvupcuwvajs6pz859yawv2awrjmh3mj4xzsuk9qfh
```
After the account is derived forc-wallet prints wallet address. Should we make it like:

```
Wallet created successfully. Deriving first account
Please enter your wallet password to derive account 0: 
Wallet address: fuel1dy60cz4kzxxzexdaqkvupcuwvajs6pz859yawv2awrjmh3mj4xzsuk9qfh
Account derived successfully.
```